server/api/middlewares/inputValidation/newArticle.js

```diff
@@ -0,0 +1,13 @@
+import Joi from 'joi';
+import newArticleSchema from '../../requestSchemas/newArticle';
+
+export const newArticleValidator = (req, res, next) => {
+  Joi.validate(req.body, newArticleSchema)
+    .then(() => {
+      next();
+    })
+    .catch((error) => {
+      console.log(error);
```

Comment:
@Princesso Please remove all `console.log` statement on line `10`. If you have to log the errors make use of the logger tool integrated to the project. Sync with @oyedejipeace for guidance on this.

Reply:
I have done this, thanks.